Model/Method/Billink.php

```diff
@@ -811,11 +807,8 @@ public function getRequestShippingData($payment)
         }
 
         $streetFormat    = $this->formatStreet($shippingAddress->getStreet());
-        $category = $this->_scopeConfig->getValue(
-            static::BUSINESS_METHOD,
-            ScopeInterface::SCOPE_STORE
-        );
-        $category = ($category == static::BUSINESS_METHOD_B2B) ? 'B2B' : 'B2C';
+        
+        $category = (strlen($shippingAddress->getCompany()) <= 0) ? 'B2C' : 'B2B';
```

Comment:
why to use strlen there to check empty string or not ?

Reply:
fixed